### PR TITLE
Change detail and log window from readonly to non-editable to facilitate reading

### PR DESCRIPTION
**Pull request by subzero79**

### CODE REVIEW

File: var/www/openmediavault/js/omv/module/admin/service/docker/ContainerGrid.js [hunk 1]
```diff
@@ -618,17 +620,16 @@ Ext.define("OMV.module.admin.service.docker.ContainerGrid", {
...
+                    editable: false,
                     height: 620,
-                    readOnly: true,
-                    cls: "x-form-textarea-monospaced"
-//                    fieldStyle: {
-  //                      fontFamily: "courier",
-    //                    fontSize: "12px"
-      //              }
+                    cls: "x-form-textarea-monospaced",
+                    fieldStyle: {
```
Comment (votdev) [inline var/www/openmediavault/js/omv/module/admin/service/docker/ContainerGrid.js]: Is there any special reason why you override the default font?

Comment (subzero79) [reply var/www/openmediavault/js/omv/module/admin/service/docker/ContainerGrid.js]: Not really i copied the change from a very old fix when zfs plugin was in development.  Now that you mention it looks much good without monospaced.

Comment (votdev) [reply var/www/openmediavault/js/omv/module/admin/service/docker/ContainerGrid.js]: The x-form-textarea-monospaced is good, but i'm wondering about the fieldStyle thingy.

Comment (nicjo814) [reply var/www/openmediavault/js/omv/module/admin/service/docker/ContainerGrid.js]: I can't really remember why this was introduced I'm afraid.

Comment (subzero79) [reply var/www/openmediavault/js/omv/module/admin/service/docker/ContainerGrid.js]: I will submit another PR with monospaced cls and no override

**Pull request merged**